From 07066a381e72282564c593d4cef04d6899f75128 Mon Sep 17 00:00:00 2001
From: Mohamad Chaarawi <mohamad.chaarawi@intel.com>
Date: Mon, 19 Aug 2019 17:17:49 +0000
Subject: [PATCH] comment out unsupported tests.

Signed-off-by: Mohamad Chaarawi <mohamad.chaarawi@intel.com>
---
 testpar/testphdf5.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/testpar/testphdf5.c b/testpar/testphdf5.c
index 69b66aefc..ab9914ff9 100644
--- a/testpar/testphdf5.c
+++ b/testpar/testphdf5.c
@@ -379,6 +379,8 @@ int main(int argc, char **argv)
       "extendible dataset independent write #2", PARATESTFILE);
     AddTest("selnone", none_selection_chunk, NULL,
             "chunked dataset with none-selection", PARATESTFILE);
+/* MSC - serial means not MPI-IO, not in DAOS tier */
+#ifdef DAOS_UNSUPPORTED
     AddTest("calloc", test_chunk_alloc, NULL,
       "parallel extend Chunked allocation on serial file", PARATESTFILE);
     AddTest("fltread", test_filter_read, NULL,
@@ -388,6 +390,7 @@ int main(int argc, char **argv)
     AddTest("cmpdsetr", compress_readAll, NULL,
       "compressed dataset collective read", PARATESTFILE);
 #endif /* H5_HAVE_FILTER_DEFLATE */
+#endif
 
     ndsets_params.name = PARATESTFILE;
     ndsets_params.count = ndatasets;
@@ -532,6 +538,8 @@ int main(int argc, char **argv)
     AddTest((mpi_size < 2)? "-fiodc" : "fiodc", file_image_daisy_chain_test, NULL,
             "file image ops daisy chain", NULL);
 
+/* MSC - not atomicity support in DAOS ADIO. */
+#ifdef DAOS_UNSUPPORTED
     if((mpi_size < 2)&& MAINPROCESS ) {
 	printf("Atomicity tests need at least 2 processes to participate\n");
 	printf("8 is more recommended.. Atomicity tests will be skipped \n");
@@ -543,6 +551,7 @@ int main(int argc, char **argv)
         AddTest("atomicity", dataset_atomicity, NULL,
                 "dataset atomic updates", PARATESTFILE);
     }
+#endif
 
     AddTest("denseattr", test_dense_attr, NULL,
 	    "Store Dense Attributes", NULL);
