From 5cc79a0e878c96dc38a1b0ae48fabeae346a7dec Mon Sep 17 00:00:00 2001
From: Maureen Jean <mjean@boro-69.boro.hpdd.intel.com>
Date: Wed, 15 Jul 2020 20:43:28 +0000
Subject: [PATCH] daos testphdf5

---
 testpar/testphdf5.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/testpar/testphdf5.c b/testpar/testphdf5.c
index b89c790..9b0285c 100644
--- a/testpar/testphdf5.c
+++ b/testpar/testphdf5.c
@@ -379,6 +379,8 @@ int main(int argc, char **argv)
             "extendible dataset independent write #2", PARATESTFILE);
     AddTest("selnone", none_selection_chunk, NULL,
             "chunked dataset with none-selection", PARATESTFILE);
+/* MSC - serial means not MPI-IO, not in DAOS tier */
+#ifdef DAOS_UNSUPPORTED
     AddTest("calloc", test_chunk_alloc, NULL,
             "parallel extend Chunked allocation on serial file", PARATESTFILE);
     AddTest("fltread", test_filter_read, NULL,
@@ -388,6 +390,7 @@ int main(int argc, char **argv)
     AddTest("cmpdsetr", compress_readAll, NULL,
             "compressed dataset collective read", PARATESTFILE);
 #endif /* H5_HAVE_FILTER_DEFLATE */
+#endif /* DAOS_UNSUPPORTED */
 
     AddTest("zerodsetr", zero_dim_dset, NULL,
             "zero dim dset", PARATESTFILE);
@@ -419,8 +422,11 @@ int main(int argc, char **argv)
 #else
     HDprintf("big dataset test will be skipped on Windows (JIRA HDDFV-8064)\n");
 #endif
+/** MSC - default fill value of 0 does not work with DAOS */
+#ifdef DAOS_UNSUPPORTED
     AddTest("fill", dataset_fillvalue, NULL,
             "dataset fill value", PARATESTFILE);
+#endif /* DAOS_UNSUPPORTED */
 
     AddTest("cchunk1",
             coll_chunk1,NULL, "simple collective chunk io",PARATESTFILE);
@@ -532,6 +538,9 @@ int main(int argc, char **argv)
     AddTest((mpi_size < 2)? "-fiodc" : "fiodc", file_image_daisy_chain_test, NULL,
             "file image ops daisy chain", NULL);
 
+/* MSC - not atomicity support in DAOS ADIO. */
+#ifdef DAOS_UNSUPPORTED
+
     if((mpi_size < 2)&& MAINPROCESS ) {
         HDprintf("Atomicity tests need at least 2 processes to participate\n");
         HDprintf("8 is more recommended.. Atomicity tests will be skipped \n");
@@ -543,6 +552,7 @@ int main(int argc, char **argv)
         AddTest("atomicity", dataset_atomicity, NULL,
                 "dataset atomic updates", PARATESTFILE);
     }
+#endif /* DAOS_UNSUPPORTED */
 
     AddTest("denseattr", test_dense_attr, NULL,
             "Store Dense Attributes", PARATESTFILE);
-- 
1.8.3.1

