diff --git a/tools/src/h5ls/h5ls.c b/tools/src/h5ls/h5ls.c
index a27a8cc..f427d6b 100644
--- a/tools/src/h5ls/h5ls.c
+++ b/tools/src/h5ls/h5ls.c
@@ -2299,7 +2299,7 @@ list_lnk(const char *name, const H5L_info2_t *linfo, void *_iter)
                 iter->symlink_list->dangle_link = TRUE;
 
             h5tools_str_append(&buffer, "Soft Link {");
-            h5tools_str_append(&buffer, buf);
+            h5tools_str_append(&buffer, " %s", buf);
             h5tools_str_append(&buffer, "}");
             h5tools_render_element(rawoutstream, info, &ctx, &buffer, &curr_pos, (size_t)info->line_ncols,
                                    (hsize_t)0, (hsize_t)0);
@@ -2367,11 +2367,11 @@ list_lnk(const char *name, const H5L_info2_t *linfo, void *_iter)
                 goto done;
 
             h5tools_str_append(&buffer, "External Link {");
-            h5tools_str_append(&buffer, filename);
+            h5tools_str_append(&buffer, " %s", filename);
             h5tools_str_append(&buffer, "/");
             if (*path != '/')
                 h5tools_str_append(&buffer, "/");
-            h5tools_str_append(&buffer, path);
+            h5tools_str_append(&buffer, " %s", path);
             h5tools_str_append(&buffer, "}");
             h5tools_render_element(rawoutstream, info, &ctx, &buffer, &curr_pos, (size_t)info->line_ncols,
                                    (hsize_t)0, (hsize_t)0);

diff --git a/tools/lib/h5tools_str.c b/tools/lib/h5tools_str.c
index 1fb2c68..5e6c9dd 100644
--- a/tools/lib/h5tools_str.c
+++ b/tools/lib/h5tools_str.c
@@ -1051,7 +1051,7 @@ h5tools_str_sprint(h5tools_str_t *str, const h5tool_format_t *info, hid_t contai
 
                 H5TOOLS_DEBUG("H5T_ENUM");
                 if (H5Tenum_nameof(type, vp, enum_name, sizeof enum_name) >= 0)
-                    h5tools_str_append(str, h5tools_escape(enum_name, sizeof(enum_name)));
+                    h5tools_str_append(str, "%s", h5tools_escape(enum_name, sizeof(enum_name)));
                 else {
                     size_t i;
                     if (1 == nsize)
